packages/create-turbo/src/commands/create/prompts.ts

```diff
@@ -98,17 +55,13 @@ export async function packageManager({
       // prompt for package manager if it wasn't provided as an argument, or if it was
       // provided, but isn't available (always allow npm)
       !packageManager ||
-      (packageManager as PackageManager) !== "npm" ||
-      !Object.keys(availablePackageManagers).includes(packageManager),
+      !availablePackageManagers?.[packageManager as PackageManager]?.available,
     choices: ["npm", "pnpm", "yarn"].map((p) => ({
       name: p,
       value: p,
-      disabled:
-        // npm should always be available
-        p === "npm" ||
```

Comment:
did we lose this logic?

Reply:
Yep, but it was intentional. This was added in an attempt to patch another bug. That bug has [since been fixed](https://github.com/vercel/turbo/pull/4603/files) so we no longer need to allow list here.